doc/release-notes.md

```diff
@@ -90,6 +90,9 @@ Low-level RPC changes
   * `getmininginfo`
 - The wallet RPC `getreceivedbyaddress` will return an error if called with an address not in the wallet.
 
+Changed command-line options
+-----------------------------
+- `-debuglogfile=<file>` can be used to specify an alternative debug logging file.
```

Comment:
Can you update the text to say that this can be absolute or relative (just like you have for the help text: `this can be an absolute path or a path relative to the data directory`)?

Reply:
The release notes are not documentation. This minor change does not warrant a long write-up in the release notes. If anyone wants to have documentation they can just look at bitcoind's help which is documentation.